src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -138,24 +140,28 @@ public int getValue(){
     private static final String R_SCRIPT = "insertSizeHistogram.R";
 
     @Override
-    protected void runTool(final JavaSparkContext ctx) {
+    public boolean requiresReads(){return true;}
```

Comment:
Minor style point but I think we should leave spaces around curly braces ie " { return true; }"